CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   the alias value. Crate an alias with 
```

Comment:
```suggestion
  the alias value. Create an alias with 
```
Typo is not part of your review. Appreciate you fixing it though.

Reply:
Nice catch!